website/static/files/build_orca_inventory_refined.sql

```diff
@@ -0,0 +1,140 @@
+    (
+      provider_id     text NOT NULL
+    , collection_id   text NOT NULL
+    , CONSTRAINT PK_provider_collection_xref PRIMARY KEY(provider_id,collection_id)
+    , CONSTRAINT provider_collection_fk FOREIGN KEY(provider_id) REFERENCES providers(provider_id)
+    , CONSTRAINT collection_provider_fk FOREIGN KEY(collection_id) REFERENCES collections(collection_id)
+    );
+
+    COMMENT ON TABLE provider_collection_xref
+      IS 'Cross refrence table that ties a collection and provider together and resolves the many to many relationship.';
+    COMMENT ON COLUMN provider_collection_xref.provider_id
+      IS 'Provider ID from the providers table.';
+    COMMENT ON COLUMN provider_collection_xref.collection_id
+      IS 'Collection ID from the collections table.';
+
+
+    -- Create the Granules table
+    CREATE TABLE IF NOT EXISTS granules
+    (
+      id                  int8 NOT NULL
+    , collection_id       text NOT NULL
+    , cumulus_granule_id  text NOT NULL
+    , execution_id  	  text NOT NULL
+    , ingest_time         timestamp with time zone NOT NULL
+    , last_update         timestamp with time zone NOT NULL
+    , CONSTRAINT PK_granules PRIMARY KEY(id)
+    , CONSTRAINT UNIQUE_granules UNIQUE (collection_id, cumulus_granule_id)
+    , CONSTRAINT collection_granule_fk FOREIGN KEY(collection_id) REFERENCES collections(collection_id)
+    );
+
+    COMMENT ON TABLE granules
+      IS 'Granules that are in the ORCA archive holdings.';
+    COMMENT ON COLUMN granules.id
+      IS 'Internal orca granule ID pseudo key';
+    COMMENT ON COLUMN granules.collection_id
+      IS 'Collection ID from Cumulus that refrences the Collections table.';
+    COMMENT ON COLUMN granules.cumulus_granule_id
+      IS 'Granule ID from Cumulus';
+    COMMENT ON COLUMN granules.execution_id
+      IS 'Step function execution ID from AWS';
+    COMMENT ON COLUMN granules.ingest_time
+      IS 'Date and time the granule was originally ingested into ORCA.';
+    COMMENT ON COLUMN granules.last_update
+      IS 'Last time the data for the granule was updated. This generally will coincide with a duplicate or a change to the underlying data file.';
+
+    -- Create the Files table
+    CREATE TABLE IF NOT EXISTS files
+    (
+      id                          int8 NOT NULL
+    , granule_id                  int8 NOT NULL
+    , name                        text NOT NULL
+    , orca_archive_location       text NOT NULL
+    , cumulus_archive_location    text NOT NULL
+    , key_path                    text NOT NULL
+    , ingest_time                 timestamp with time zone NOT NULL
+    , etag                        text NOT NULL
+    , version                     text NOT NULL
+    , size_in_bytes               int8 NOT NULL
+    , hash                        text NULL
+    , hash_type                   text NULL
+    , CONSTRAINT PK_files PRIMARY KEY(id)
+    , CONSTRAINT UNIQUE_files_1 UNIQUE (cumulus_archive_location, key_path)
```

Comment:
Change name `UNIQUE_files_1` to `UNIQUE_cumulus_archive_location_key_path` to match what is in the db_deploy orca_sql.py

Reply:
Side branch